sql/2020/03_Markup/README.md

```diff
@@ -0,0 +1,164 @@
+# Guide 
+
+## file naming
+file names start with the table being queried. Next is the custom metric if appropriate. Then "_by_" followed by how the query is grouped. Finally anything more specific needed.
+
+Queries contain one or more metrics that match the queries grouping structure. 
```

Comment:
I'm torn on this. I know it saves on quota, like a query carpool, but it also makes the queries more overwhelming to understand how a single metric is evaluated.

In #1138 I laid out some ideas for directly linking figures to corresponding SQL and results. Carpooling would map many figures to the same SQL file, and it won't be clear where the stats are coming from.

If the obstacle isn't writing each individual query, but rather running them and exceeding quota, then we have some ways to mitigate that. Either with more coupon codes or someone with access to the HTTP Archive billing account can run them for you. Let me know if that could work for you.

Another benefit to fragmentation is that the SQL files become more browsable, if each file is named according to its metric.

Reply:
I really don't want to spend even more hours (And don't have them) splitting this all out, having to re-test, deal with the massively multiplied maintenance issue and then have to go through the data gathering process many more times than I need to. In some cases my job will be multiplied maybe 30xs and so will the bill. And I have to do this over two chapters where their 20 odd queries each will bloat to over 100.

I'd also argue about it making them more overwhelming. Dealing with 100 queries that vary by small amounts would be more overwhelming than seeing the same pattern used in the one place. And making one fix or improvement would take a lot longer and would most likely cause code to get out of sync and broken. 

I personally don't see how it makes the sheet more complex. I'd personally would prefer working with groups of results in one tab, instead of 100s of separate tabs. See my first example for markup :

https://docs.google.com/spreadsheets/d/1Ta7amoUeaL4pILhWzH-SCzMX9PsZeb1x_mwrX2C4eY8/edit?ts=5f4e700c#gid=150962402

If one tab gets too congested the data could be split into a new tab with smaller groups of info. I've been pushing the team to look at what I have done so far, to try and find out if they are OK with the structure. No feedback yet.

For both chapters I've put a lot of effort in directing me and the the others to where the data is. Noting down the script and field name in their documentation and mine. In this case I even created numbered labels for the fields . It would be nice if we could create links to a specific tab or cell on a sheet 👍